var/spack/repos/builtin/packages/fms/package.py

```diff
@@ -29,6 +29,18 @@ class Fms(CMakePackage):
     version('2020.04.02', sha256='bd6ce752b1018d4418398f14b9fc486f217de76bcbaaf2cdbf4c43e0b3f39f69')
     version('2020.04.01', sha256='2c409242de7dea0cf29f8dbf7495698b6bcac1eeb5c4599a728bdea172ffe37c')
 
+    # DH* 20220602
+    # These versions were adapated by JCSDA and are only meant to be
+    # used temporarily, until the JCSDA changes have found their way
+    # back into the official repository.
+    # The following commit corresponds to branch='feature/no-openmp-option_default_on'
+    version('release-jcsda', commit="6ed89b23e3dc7b8d74191f92760a9487de93a85b", no_cache=True)
```

Comment:
Maybe we should add `preferred = False`, or is that the default?

Reply:
String versions are lower than numbers and the newest version is preferred which is 2022.02.

Reply:
> String versions are lower than numbers and the newest version is preferred which is 2022.02, assuming it's not in `packages.yaml`.

That's great, thanks for confirming.